### ISSUE TITLE
Create receiver for CDC-ELIMS-HL7 org in STAGING environment

### ISSUE THREAD

**Issue by arnejduranovic** [CDCgov/prime-reportstream#10200]
### User Story

As the CDC-ELIMS-HL7 organization, I need the ability to receive all HL7v2 ELR (ORU-R01) messages flowing through RS that have the "elims-elr" topic assigned.

### Description/Use Case

We need to configure RS:
1. Create a receiver under the CDC-ELIMS-HL7 org in staging that can receive any HL7v2 ELR message with the "elims-elr" topic
2. Setup transport protocol with CDC-ELIMS in staging and test they can receive a message. **ELIMS would like to use SFTP WITH our Azure Blob Store**, so we will need to contact them for the info.

### Risks/Impacts/Considerations

- CDC ELIMS would like to receive the exact same message that goes to a particular STLT. This means that we CANNOT use any kind of transforms until a solution is figured out that lets us send an exact copy of the message a STLT received. This is because if we set up a special transform for a STLT, then the cdc receiver will need that same exact transform configured otherwise the message they and the STLT receive will be different and this is untenable due to having to support more than one STLT.

### Dev Notes

Request for connectivity from ELIMS:
1. We can connect Rhapsody to Azure Storage BLOBS using sFTP client comm points
  - You can configure Azure Storage BLOBS to make their data available through sFTP interfaces
  - Each Azure Storage BLOB could be considered sFTP storage and Rhapsody could interact with it the same as it interacts with any sFTP server host
  - This approach would only work if RS has turned on sFTP services for the Storage BLOB

### Acceptance Criteria

- [x] receiver created and configured
- [x] transport protocol configured (SFTP with Azure Blob)
- [x] Tested receiver can receive any ORU-R01 message under the "elims-elr" topic

**Comment by bishoyayoub** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-6182b02547c1130010f459db/issues/CDCgov/prime-reportstream/10200?planning-poker) with Zenhub @arnejduranovic @JessicaWNava @JFU-NAVA-PBC @jimmyfagan @luis-pabon-tf @thetaurean

**Comment by avnieldravid** [comment]
we plan on discussing this today at 3pm

**Comment by avnieldravid** [comment]
Email sent to ELIMS team on 7/12

**Comment by avnieldravid** [comment]
We will setup ELIMS to receive messages with the REST protocol .

**Comment by avnieldravid** [comment]
@arnejduranovic this should be unblocked now

**Comment by avnieldravid** [comment]
We actually still need the details of the endpoint and instructions on how to connect.  so still blocked, until Christ Taylor is back in the office next week.

**Comment by MauriceReeves-usds** [comment]
@avnieldravid A receiver could be set up for ELIMS in staging without knowing all the information for Rhapsody and messages that are sent to them would appear in the UI, or land in blob storage. Then it's a matter of just doing a settings update to add the REST endpoint, etc.

I also have some doubts about sending the message via REST. Maybe it would be better to set them up to receive via blob storage and their Rhapsody instance could then pull message content from the store like we do with HHSProtect and DCIPHER?

**Comment by arnejduranovic** [comment]
@MauriceReeves-usds 

1. Yes, we already have a CDC ELIMS receiver on the staging environment with everything set up except for the SFTP transport protocol info. 
2. I agree, I updated the story a while ago to indicate we will be using SFTP instead of REST. The ELIMS team said they would prefer SFTP and were comfortable with it and knew how to connect with it, so that's why we chose that over blob storage.
